### PR TITLE
shifted imports & added blank Lines

### PR DESCRIPTION
**Pull request by ranbir7**
Please check if it is correct....

### CODE REVIEW

File: autoprof/fit/gradient.py [hunk 1]
```diff
@@ -1,8 +1,10 @@
+#changed according to PEP 8
```
Comment (ConnorStoneAstro) [inline autoprof/fit/gradient.py]: No need to include a comment about PEP8

File: autoprof/models/nonparametric_model.py [hunk 2]
```diff
@@ -6,6 +6,7 @@
 from .ray_model import Ray_Galaxy
 from .wedge_model import Wedge_Galaxy
 from ._shared_methods import parametric_initialize
+
 import numpy as np
 import torch
```
Comment (ConnorStoneAstro) [inline autoprof/models/nonparametric_model.py]: numpy and torch should be imported above the local imports

**Comment by ranbir7** [comment]
whats happening here?

**Comment by ConnorStoneAstro** [comment]
Those are new code coverage tests that show how much of the code is being checked by the tests in the `tests\` directory. Your code passed the unit tests, so it is good. You can ignore the coverage warnings, they are just letting me know that more code has been added without adding more tests.

File: autoprof/utils/initialize/construct_psf.py [hunk 3]
```diff
@@ -1,6 +1,4 @@
-import numpy as np
-from .center import Lanczos_peak, center_of_mass, GaussianDensity_Peak
-from ..interpolate import shift_Lanczos_np, point_Lanczos
+
```
Comment (ConnorStoneAstro) [inline autoprof/utils/initialize/construct_psf.py:2]: Looks like some lines have been deleted here, can they be added back in?

**Review by ConnorStoneAstro — COMMENTED** [review]
This looks great! I've added some minor comments to address. Again, you can ignore the code coverage checks, those are just for me to track how much code is being tested by the unit tests, they don't actually indicate anything wrong with the code.

File: autoprof/models/nonparametric_model.py (hunk 2 above)
Comment (ranbir7) [reply autoprof/models/nonparametric_model.py]: I'll be solving each one soon 
Thank God the problem was something else
I thought my whole night's work went down the drain!

File: autoprof/utils/initialize/construct_psf.py (hunk 3 above)
Comment (ranbir7) [reply autoprof/utils/initialize/construct_psf.py:2]: yes for sure

**Review by ConnorStoneAstro — APPROVED** [review]
Everything looks good, great work on this!!!

**Comment by ConnorStoneAstro** [comment]
This pull request is listed as a draft pull request, are you still working on it or can it be merged?

**Comment by ranbir7** [comment]
It was before 
now I have made it ready for review

**Comment by ranbir7** [comment]
will u be closing the issue?

**Comment by ConnorStoneAstro** [comment]
The PR is merged so it is now considered closed. Issue #23 which is the connected issue for this PR will remain open since there are still some files with imports out of order. This was a big step towards #23 though!

**Comment by ranbir7** [comment]
okay okay 
tell me all those files..

**Comment by ConnorStoneAstro** [comment]
There are a number, for example `autoprof/image/model_image.py` or `autoprof/image/target_image.py` and `autoprof/model/flatsky_model.py` or `autoprof/model/core_model.py`. YOu have already done more than half of them though, which is really great!

**Comment by ranbir7** [comment]
okayy
I'll be looking into them!
